frontend/model/notifications/utils.js

```diff
@@ -96,8 +92,8 @@ export function compareOnPriority (notificationA: Notification, notificationB: N
     // However, under certain conditions we might want to keep the already read one instead.
     // Note: use a condition array so that it is easy to add/remove conditions later.
     const conditions = [
-      (read, unread) => age(read) < ONE_DAY && age(unread) > FIFTEEN_DAYS,
-      (read, unread) => age(read) < TWO_DAYS && age(unread) > THREE_WEEKS
+      (read, unread) => Date.now() - read.timestamp < ONE_DAY && Date.now() - unread.timestamp > FIFTEEN_DAYS,
+      (read, unread) => Date.now() - read.timestamp < TWO_DAYS && Date.now() - unread.timestamp > THREE_WEEKS
```

Comment:
> @taoeffect However removing the age function is going to make the isExpired() function and the conditions in compareOnPriority() a bit less readable

Hmm, I agree. If you'd like to re-add it in order to use it here and below (in `isExpired`), feel free.